src/smpp_gateway/migrations/0008_mtmessage_is_transactional.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.16 on 2024-11-12 14:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smpp_gateway', '0007_momessage_error_alter_momessage_status'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
If possible, let's remove the three migration files and regenerate a single migration to add only the `priority_flag` field and change the indexes.